scriptshifter/tables/data/asian_cyrillic.yml

```diff
@@ -384,15 +384,19 @@ roman_to_script:
     "(|)": "\u0482"
     "(^)": "\u0488"
     "(')": "\u0489"
-    
+
+    # Two Less-than signs mapped to Left-pointing double angle quotation mark    
     "\u003C\u003C": "\u00AB"
+    # Two Greater-than signs mapped to Right-pointing double angle quotation mark    
     "\u003E\u003E": "\u00BB"

```

Comment:
Is this a *transliteration* task, specific to Cyrillic, or a *normalization* task that applies elsewhere? 

We have now a `normalize` section available in the configuration. It is only available in S2R at the moment but I could implement it for the other direction too.